### DISCUSSION TITLE
Mini D1 - Cannot upgrade from 8.34 to 8.35

### DISCUSSION THREAD

**Discussion by anthonws** [technyon/nuki_hub#418] [Q&A]
Hello,

I've tried everything, from different browsers to different power cables/PSUs, but I am not able to upgrade my Mini D1 ESP32 from 8.34 to 8.35.

Any hints? Maybe it needs retirement? :P

> Nuki Hub version: 8.34
> run: true
> confVersion: 834
> deviceId: 2895845103
> deviceIdOp: 2895845103
> nukiId: ***
> nukidOp: ***
> mqttbroker: 192.168.25.9
> mqttport: 1883
> mqttuser: 
> mqttpass: 
> mqttlog: false
> checkupdates: true
> lockena: true
> lockpin: 1
> mqttpath: nuki
> openerena: true
> openerpin: 1
> openercont: false
> mqttoppath: nukiopener
> maxkpad: 
> opmaxkpad: 
> mqttca: 
> mqttcrt: 
> mqttkey: 
> hassdiscovery: homeassistant
> hassConfigUrl: 
> dhcpena: true
> ipaddr: 
> ipsub: 
> ipgtw: 
> dnssrv: 
> nwhw: 1
> nwwififb: false
> rssipb: 60
> hostname: nukihub
> nettmout: 60
> restdisc: true
> rstbcn: 60
> lockStInterval: 1800
> configInterval: 3600
> batInterval: 1800
> kpInterval: 1800
> kpCntrlEnabled: true
> kpInfoEnabled: false
> aclLckOpn: 
> tcCntrlEnabled: false
> tcInfoEnabled: false
> confLckBasAcl: 
> confLckAdvAcl: 
> confOpnBasAcl: 
> confOpnAdvAcl: 
> accLvl: 0
> regAsApp: false
> nrRetry: 0
> rtryDelay: 100
> crdusr: 
> crdpass: 
> pubAuth: false
> pubdbg: false
> prdtimeout: 60
> hasmac: false
> macb0: 
> macb1: 
> macb2: 
> latest: 8.35
> MQTT connected: Yes
> Lock firmware version: 
> Lock hardware version: 
> Lock paired: Yes
> Lock PIN set: Yes
> Lock has door sensor: No
> Lock has keypad: No
> Lock ACL (Lock): Allowed
> Lock ACL (Unlock): Allowed
> Lock ACL (Unlatch): Allowed
> Lock ACL (Lock N Go): Allowed
> Lock ACL (Lock N Go Unlatch): Allowed
> Lock ACL (Full Lock): Allowed
> Lock ACL (Fob Action 1): Allowed
> Lock ACL (Fob Action 2): Allowed
> Lock ACL (Fob Action 3): Allowed
> Lock config ACL (Name): Disallowed
> Lock config ACL (Latitude): Disallowed
> Lock config ACL (Longitude): Disallowed
> Lock config ACL (Auto Unlatch): Disallowed
> Lock config ACL (Pairing enabled): Allowed
> Lock config ACL (Button enabled): Allowed
> Lock config ACL (LED flash enabled): Allowed
> Lock config ACL (LED brightness): Disallowed
> Lock config ACL (Timezone offset): Disallowed
> Lock config ACL (DST mode): Disallowed
> Lock config ACL (Fob Action 1): Disallowed
> Lock config ACL (Fob Action 2): Disallowed
> Lock config ACL (Fob Action 3): Disallowed
> Lock config ACL (Single Lock): Allowed
> Lock config ACL (Advertising Mode): Disallowed
> Lock config ACL (Timezone ID): Disallowed
> Lock config ACL (Unlocked Position Offset Degrees): Disallowed
> Lock config ACL (Locked Position Offset Degrees): Disallowed
> Lock config ACL (Single Locked Position Offset Degrees): Disallowed
> Lock config ACL (Unlocked To Locked Transition Offset Degrees): Disallowed
> Lock config ACL (Lock n Go timeout): Disallowed
> Lock config ACL (Single button press action): Disallowed
> Lock config ACL (Double button press action): Disallowed
> Lock config ACL (Detached cylinder): Disallowed
> Lock config ACL (Battery type): Disallowed
> Lock config ACL (Automatic battery type detection): Disallowed
> Lock config ACL (Unlatch duration): Disallowed
> Lock config ACL (Auto lock timeout): Disallowed
> Lock config ACL (Auto unlock disabled): Allowed
> Lock config ACL (Nightmode enabled): Disallowed
> Lock config ACL (Nightmode start time): Disallowed
> Lock config ACL (Nightmode end time): Disallowed
> Lock config ACL (Nightmode auto lock enabled): Disallowed
> Lock config ACL (Nightmode auto unlock disabled): Disallowed
> Lock config ACL (Nightmode immediate lock on start): Disallowed
> Lock config ACL (Auto lock enabled): Allowed
> Lock config ACL (Immediate auto lock enabled): Disallowed
> Lock config ACL (Auto update enabled): Allowed
> Opener firmware version: 
> Opener hardware version: 
> Opener paired: No
> Opener PIN set: -
> Opener has keypad: No
> Opener ACL (Activate Ring-to-Open): Allowed
> Opener ACL (Deactivate Ring-to-Open): Allowed
> Opener ACL (Electric Strike Actuation): Allowed
> Opener ACL (Activate Continuous Mode): Allowed
> Opener ACL (Deactivate Continuous Mode): Allowed
> Opener ACL (Fob Action 1): Allowed
> Opener ACL (Fob Action 2): Allowed
> Opener ACL (Fob Action 3): Allowed
> Opener config ACL (Name): Disallowed
> Opener config ACL (Latitude): Disallowed
> Opener config ACL (Longitude): Disallowed
> Opener config ACL (Pairing enabled): Disallowed
> Opener config ACL (Button enabled): Allowed
> Opener config ACL (LED flash enabled): Allowed
> Opener config ACL (Timezone offset): Disallowed
> Opener config ACL (DST mode): Disallowed
> Opener config ACL (Fob Action 1): Disallowed
> Opener config ACL (Fob Action 2): Disallowed
> Opener config ACL (Fob Action 3): Disallowed
> Opener config ACL (Operating Mode): Disallowed
> Opener config ACL (Advertising Mode): Disallowed
> Opener config ACL (Timezone ID): Disallowed
> Opener config ACL (Intercom ID): Disallowed
> Opener config ACL (BUS mode Switch): Disallowed
> Opener config ACL (Short Circuit Duration): Disallowed
> Opener config ACL (Eletric Strike Delay): Disallowed
> Opener config ACL (Random Electric Strike Delay): Disallowed
> Opener config ACL (Electric Strike Duration): Disallowed
> Opener config ACL (Disable RTO after ring): Disallowed
> Opener config ACL (RTO timeout): Disallowed
> Opener config ACL (Doorbell suppression): Disallowed
> Opener config ACL (Doorbell suppression duration): Disallowed
> Opener config ACL (Sound Ring): Disallowed
> Opener config ACL (Sound Open): Disallowed
> Opener config ACL (Sound RTO): Disallowed
> Opener config ACL (Sound CM): Disallowed
> Opener config ACL (Sound confirmation): Disallowed
> Opener config ACL (Sound level): Allowed
> Opener config ACL (Single button press action): Disallowed
> Opener config ACL (Double button press action): Disallowed
> Opener config ACL (Battery type): Disallowed
> Opener config ACL (Automatic battery type detection): Allowed
> Network device: Built-in Wi-Fi
> Uptime: 0 minutes
> Heap: 74812
> Stack watermarks: nw: 5040, nuki: 2696, pd: 256
> Restart reason FW: NotApplicable
> Restart reason ESP: ESP_RST_BROWNOUT: Brownout reset (software or hardware)

**Comment by iranl** [comment]
Brownout reset usually means a bad/inadequate power supply.
Try  another way of powering your device (other USB power brick and/or cable)

Edit: Sorry missed that you already tried that. Did you try updating using usb/serial or only using ota? If that also doesn't work i'd try a new esp.

**Comment by anthonws** [comment]
Ended up being a faulty device. Replaced with a new model, since the AZDelivery Mini D1 were/are very unreliable. Thank you!
